docs/reference/glossary.md

```diff
@@ -2443,38 +2456,50 @@ Very Secure FTP Daemon
 :::{glossary}
 
 WAL
-Write-Ahead Logging
-    *Work in Progress*
+Write-Ahead Log
+    WAL is a fundamental technique used in databases and storage systems to ensure data durability and consistency. The basic principle is simple write what you are going to do before you do it, so if things go wrong you can always know what was happening. In other words, before any changes are made to the actual database files, the system first writes a record of what it is about to do to a sequential log file. If the system crashes mid-operation, it can replay the log during recovery to complete interrupted operations or roll them back. 
```

Comment:
```suggestion
    WAL is a fundamental technique used in databases and storage systems to ensure data durability and consistency. The basic principle is simple: Write what you are going to do before you do it, so if things go wrong you can always know what was happening. In other words, before any changes are made to the actual database files, the system first writes a record of what it is about to do to a sequential log file. If the system crashes mid-operation, it can replay the log during recovery to complete interrupted operations or roll them back.
```